sdk/loadtestservice/load-testing-rest/samples-dev/sample.ts

```diff
@@ -0,0 +1,63 @@
+import AzureLoadTesting, { AzureLoadTestingClient } from "@azure-rest/load-testing";
+import { DefaultAzureCredential } from "@azure/identity";
+import dotenv from "dotenv";
+import { createReadStream } from "fs";
+
+const readStream = createReadStream("./sample.jmx");
+dotenv.config({ path: "../../sample.env" });
```

Comment:
You don't need to add the path to `sample.env`, users will need to create .env next to the package.json and dotenv will pick it up automatically you just need `dotenv.config()`

Reply:
Done.